config/config_exec/browserstack_conf/local.conf.js

```diff
@@ -1,18 +1,12 @@
 var browserstack = require('browserstack-local');
-const config = require('config'); 
+process.env["NODE_CONFIG_DIR"] ="./config/config_env";
+let config = require('config');
 var url = config.get("web.url");
 let bs_local = new browserstack.Local();
-var globalTunnel = require('global-tunnel-ng');
-globalTunnel.initialize({
-  host: 'internetpu',
-  port: 8085,
-
-});
+credential = require("../credentials.js")
 
 exports.config = {
    specs: ["../spec/TodoSpec.js"],
-   //seleniumAddress: 'http://hub-cloud.browserstack.com/wd/hub',
-  // webDriverProxy:'http://internetpu:8085',
    jasmineNodeOpts: {
         showColors: true, // Use colors in the command line report.
         defaultTimeoutInterval: 1000000,
```

Comment:
you can have this also in the global config

Reply:
Done, Added timeout variable in credentials.js , which is common env variable file